api/resources/translations/messages-en.properties

```diff
@@ -842,6 +842,7 @@ messages.f.report_accepted_parent = {{patient_name}} ({{patient_id}}) was flagge
 messages.generic.no_provided_patient_id = External ID number support is configured, but is not found on document in the configured location.
 messages.generic.provided_patient_id_not_unique = The provided ID number is already used.
 messages.generic.provided_phone_not_valid = The provided phone is not valid.
+messages.invalid_phone= The provided phone is not valid.
```

Comment:
These two have the same content. Can't we use the same key whereve this is called?

Reply:
I saw in your app settings you had added invalid_phone hence I thought it was used somewhere. (i.e. Became a sheep) Will remove it.